layers/meta-resin-allwinner/recipes-core/images/resin-image.inc

```diff
@@ -1,3 +1,21 @@
+#
```

Comment:
Please redo the commit header.

For this commit, the header should be something like:
> resin-image.inc: <Short summary of changes here>

So the header should always reference the file being changed, not the layer (layers/meta-resin-allwinner)
This change has to be done on all the commits